frontend/pages/policies/ManagePoliciesPage/components/RemovePoliciesModal/RemovePoliciesModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import Modal from "components/modals/Modal";
+import Button from "components/buttons/Button";
+
+const baseClass = "remove-policies-modal";
+
+interface IRemovePoliciesModalProps {
+  onCancel: () => void;
+  onSubmit: (ids: any) => void; // TODO typing for mouse event?
```

Comment:
Referring to your TODO comment: It depends on how you defined the function. If you are using the event to `preventDefault` then you may need a type like `(ids: any) => (evt: React.MouseEventHandler<HTMLButtonElement>) => void`.

Also, type `ids`. Seems like a `string[]` or `number[]`.

Reply:
I'm having a similar typing issue here as in AddPolicyModal. Let's try to discuss on Monday.